dbt/models/default/default.vw_pin_history.sql

```diff
@@ -6,6 +6,12 @@ SELECT
     leg.user1 AS township_code,
     town.township_name,
     REGEXP_REPLACE(par.nbhd, '([^0-9])', '') AS nbhd,
+    -- Add pre-mailed values to cover us for the period after the rollover but
+    -- before the year's assessment cycle starts
+    vwpv.pre_mailed_class,
+    vwpv.pre_mailed_bldg,
+    vwpv.pre_mailed_land,
+    vwpv.pre_mailed_tot,
```

Comment:
The lack of these values was causing all 2025 records in the `reporting.vw_top_5_muni` view to have null values for stage and `ccao_av`. We have to add them into this source view so that we can properly pull them into `vw_top_5_muni`.